### PR TITLE
Provide System.Composition package readme

### PR DESCRIPTION
**Pull request by eNeRGy164**
Fill out details for the `System.Composition` NuGet package readme.

See https://github.com/dotnet/runtime/issues/99358

### CODE REVIEW

File: src/libraries/System.Composition/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+
```
Comment (ViktorHofer) [inline src/libraries/System.Composition/src/PACKAGE.md:2]:
```suggestion
```

**Review by ViktorHofer — COMMENTED** [review]
System.Composition is a meta package that brings in the other packages without providing an assembly. Should we have just one package readme for all the System.Composition packages or does it make sense to have a dedicated readme per Composition package?

File: src/libraries/System.Composition/src/PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+## About
```
Comment (ViktorHofer) [inline src/libraries/System.Composition/src/PACKAGE.md:3]:
```suggestion
## About
```

**Comment by ViktorHofer** [comment]
@dotnet/area-system-composition PTAL

**Review by buyaa-n — APPROVED** [review]
LGTM, thanks!

**Pull request merged**